### PR TITLE
Get the yaml path from the envvar and add the pipeline definition.

### PR DESCRIPTION
**Pull request by tmatsuo**

### CODE REVIEW

File: builder/create-dockerfile/src/GenFiles.php [hunk 1]
```diff
@@ -23,21 +23,28 @@ class GenFiles
...
+    private $appYamlPath;
+
     /**
      * Constructor allows injecting the workspace directory.
      */
     public function __construct($workspace = self::DEFAULT_WORKSPACE)
     {
         $this->workspace = $workspace;
+        $yamlPath = getenv('GAE_APPLICATION_YAML_PATH') !== false
+            ?: self::DEFAULT_YAML_PATH;
```
Comment (chingor13) [inline builder/create-dockerfile/src/GenFiles.php]: won't `$yamlPath` be true if the env variable is set?

Comment (tmatsuo) [inline builder/create-dockerfile/src/GenFiles.php]: Yes, you're right. I added a test and it should fail.

Comment (tmatsuo) [inline builder/create-dockerfile/src/GenFiles.php]: Failed as expected
```
RuntimeException: The application yaml file does not exist: /tmp/GenFilesTesteLdIFk/1
```

BTW true becomes `1`!

This time the test should pass.

**Comment by tmatsuo** [comment]
@chingor13 Thanks! Now it's passing. PTAL

**Pull request merged**